src/HttpRequestHandler.cpp

```diff
@@ -25,6 +28,33 @@ HttpResponse HttpRequestHandler::handleRequest(HttpRequest input)
 	return HttpResponse(ExceptionManager::getErrorStatus(InternalException("Something went wrong")), "");
 }
 
+bool HttpRequestHandler::findCgi(std::string uri)
+{
+	size_t found = uri.find("/cgi_bin");
+	if (found != std::string::npos)
+	{
+		bool ret = validateCgi(uri);
+		return ret;
+	}
+	return false;
+}
+
+bool HttpRequestHandler::validateCgi(std::string uri)
+{
+	std::string suffix = ".py";
+	std::string fullPath = FileHandler::getFilePath(uri);
+
+	if (access(fullPath.c_str(), F_OK) == 0)
+	{
+		int pos = fullPath.find(suffix);
```

Comment:
Should we check if the pos is npos so not found at all? would be faster than checking the compare etc substring etc even if the whole suffix is not there!

Reply:
sure